courses/machine_learning/deepdive2/structured/labs/1a_explore_data_babyweight.ipynb

```diff
@@ -13,7 +13,7 @@
     "\n",
```

Comment:
<p>discrepancy with the solution notebook also has:</p><pre class="ql-syntax" spellcheck="false">!sudo chown -R jupyter:jupyter /home/jupyter/training-data-analyst
</pre><p>and</p><pre class="ql-syntax" spellcheck="false">sudo pip freeze | grep google-cloud-bigquery==1.6.1 || \
sudo pip install google-cloud-bigquery==1.6.1
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/946/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>